dpgen/auto_test/common_equi.py

```diff
@@ -103,11 +115,11 @@ def make_equi(confs,
         relax_param['cal_setting'] = {"relax_pos": True,
                                       "relax_shape": True,
                                       "relax_vol": True}
-    elif "relax_pos" not in relax_param['cal_setting']:
+    if "relax_pos" not in relax_param['cal_setting']:
         relax_param['cal_setting']['relax_pos'] = True
-    elif "relax_shape" not in relax_param['cal_setting']:
+    if "relax_shape" not in relax_param['cal_setting']:
         relax_param['cal_setting']['relax_shape'] = True
-    elif "relax_vol" not in relax_param['cal_setting']:
+    if "relax_vol" not in relax_param['cal_setting']:
```

Comment:
@kevinwenminion plz check if the changes are fixing a bug in the code.

Reply:
I don't think these three "if" can be in parallel with the previous "if" for 'cal_setting' in relax_param or not. If 'cal_setting' is not in relax_param, which means the first "if" is true, there is no meaning to go through the next three "if".

Reply:
@kevinwenminion 
It seems, these codes are doing initialization for the value of 'relax_pos', 'relax_shape' and 'relax_vol' in  relax_param['cal_setting']. 
We should set the value for each key.
When 'cal_setting' is in relax_param, but only set one or none of 'relax_pos', 'relax_shape' and 'relax_vol', there will leads to at least one parameter has no value.

Reply:
Thanks for pointing this out. Maybe we could revise the code like this:

if 'cal_setting' not in relax_param:
        relax_param['cal_setting'] = {"relax_pos": True,
                                      "relax_shape": True,
                                      "relax_vol": True}
else:
    if "relax_pos" not in relax_param['cal_setting']:
        relax_param['cal_setting']['relax_pos'] = True
    if "relax_shape" not in relax_param['cal_setting']:
        relax_param['cal_setting']['relax_shape'] = True
    if "relax_vol" not in relax_param['cal_setting']:
        relax_param['cal_setting']['relax_vol'] = True

Reply:
Ok, your advise is good, I will modify the code.